src/commands/moderation/clearwarns.ts

```diff
@@ -0,0 +1,61 @@
+import { CommandCategories, CommandDefinition, createErrorEmbed } from '../index';
+import { createEmbed } from '../../lib/embed';
+import Warn from '../../schemas/warn';
+
+export const clearwarns: CommandDefinition = {
+    names: ['clearwarns'],
+    description: 'Clears the specified number of warns a user has. `Arguments: <id> <count>`',
+    category: CommandCategories.MODERATION,
+    permissions: ['KickMembers'],
```

Comment:
I don't know the DFD permission scheme, but I will recommend setting this to user management (timeout) rather than kick/ban.